resources/views/settings/subscription/index.blade.php

```diff
@@ -13,6 +13,20 @@
 
 @section('content')
     @include('partials.errors')
+    @if(session('sub_value'))
+        <div class="text-center">
+            <p>  
+                <a  href="{{ route('settings.boost') }}" class="btn btn-primary btn-lg mr-4" target="blank">
+                    <i class="fa-solid fa-rocket mr-1" aria-hidden="true"></i>     
+                    {{ __('settings/boosters.ready.title') }}
+                </a>
+                <a  href="{{ route('settings.apps') }}" class="btn btn-primary btn-lg ml-4" target="blank">
```

Comment:
Missing the check "If they don't have Discord synced already" ;-)
This can be checked on the user's `discord()` method if it's not empty